client/question/services/FeedbackGeneratorService.js

```diff
@@ -323,12 +345,10 @@ tie.factory('FeedbackGeneratorService', [
           throw new Error(['Unrecognized prereq check failure type ',
             'in getPrereqFailureFeedback().'].join());
         }
-
```

Comment:
Just a note -- I want to caution about deleting all these lines, here and elsewhere in this file (see the diff in "Files Changed" on github). They were there in this file to separate blocks of code from each other so that it's easier to read / more consumable.

I'm not strongly against their deletion (it doesn't really matter either way, code-wise), but if it's unintentional, please consider reinstating.

Reply:
I think these were flagged by code test coverage for some reason; so, I just deleted them. I can add them back in if you'd like and hope the code test coverage doesn't flag it again (I guess we could approve/push the PR anyway if it came down to it).

Reply:
Oh, that's odd. They should not be flagged by coverage because there is nothing there to test. But I think having the line breaks is better if that's possible.